server/src/main/java/org/apache/gravitino/server/web/rest/FilesetOperations.java

```diff
@@ -168,6 +186,15 @@ public Response createFileset(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "load-fileset." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "load-fileset", absolute = true)
+  @AuthorizationExpression(
+      expression =
+          "METALAKE::READ_FILESET || CATALOG::READ_FILESET "
+              + "|| SCHEMA::READ_FILESET || FILESET::READ_FILESET "
```

Comment:
Please use any expression like ANY(SELECT_TABLE,METALAKE,CATALOG,SCHEMA,TABLE)

Please refer to https://github.com/apache/gravitino/pull/7577/files